package.json

```diff
@@ -294,6 +295,34 @@
     "watchify": "^3.11.1",
     "webpack": "^4.41.6"
   },
+  "allowScripts": {
+    "node-sass": true,
+    "optipng-bin": true,
+    "gifsicle": true,
+    "jpegtran-bin": true,
+    "scrypt": true,
+    "chromedriver": true,
+    "geckodriver": true,
+    "@sentry/cli": true,
+    "weak": false,
```

Comment:
I take it this is part of disabling `weak` in `dnode`?

Reply:
Yes indeed

Reply:
`weak`'s build process is old and weird. it seems to be indirect, doesn't have its own install hook that triggers the build, i assume it relies on `bindings@1` to get its hook called and then build weak. the indirectness makes it incompatible with `allow-scripts`. `weak`'s build is not used in prod, so we can just not build it.